syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -1036,10 +1039,23 @@ def encrypt(self, protocol="mpc", **kwargs):
 
             return x_encrypted
 
+        elif protocol.lower() == "ckks":
+            context = kwargs.get("public_key")
+            scale = kwargs.get("scale")
+            if scale is None:
+                scale = DEFAULT_CKKS_SCALE
```

Comment:
Can be simplified to `val = kwargs.get("val", "default value")` to be consistent with the current code style

Reply:
True !